packages/riipen-ui/src/components/RadioGroup.test.jsx

```diff
@@ -0,0 +1,153 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import RadioGroup from "./RadioGroup.jsx";
+import Radio from "./Radio.jsx";
+
+describe("<RadioGroup>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<RadioGroup />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<RadioGroup />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("renders given non null children", () => {
+      const child = <Radio label="one" readOnly />;
+      const child2 = <Radio label="two" readOnly />;
+
+      const wrapper = mount(
+        <RadioGroup>
+          {child}
+          {null}
+          {child2}
+        </RadioGroup>
+      );
+
+      const childrenNodes = wrapper.find("fieldset").find("Radio");
+      expect(childrenNodes.at(0).props().label).toEqual("one");
+      expect(childrenNodes.at(1).props().label).toEqual("two");
+    });
+  });
+
+  describe("error prop", () => {
+    it("displays error", () => {
+      const error = <span>an error!</span>;
+
+      const wrapper = mount(<RadioGroup error={error} />);
+
+      expect(wrapper.find("fieldset").contains(error)).toEqual(true);
+    });
+  });
+
+  describe("hint prop", () => {
+    it("renders InputLabel when hint is provided", () => {
+      const hint = "it's a hint";
+
+      const wrapper = mount(<RadioGroup hint={hint} />);
+
+      expect(wrapper.find("InputLabel")).toHaveLength(1);
+    });
+
+    it("passes given hint to InputLabel", () => {
+      const hint = "it's a hint";
+
+      const wrapper = mount(<RadioGroup hint={hint} />);
+
+      expect(wrapper.find("InputLabel").props().hint).toEqual(hint);
+    });
+  });
+
+  describe("label prop", () => {
+    it("renders InputLabel when label is provided", () => {
+      const label = "it's a label";
+
+      const wrapper = mount(<RadioGroup label={label} />);
+
+      expect(wrapper.find("InputLabel")).toHaveLength(1);
+    });
+
+    it("passes given label to InputLabel as children", () => {
+      const label = "it's a label";
+
+      const wrapper = mount(<RadioGroup label={label} />);
+
+      expect(wrapper.find("InputLabel").props().children).toEqual(label);
+    });
```

Comment:
Since these tests use the same setup, is it worthwhile to combine them?

Something like...
"given label is rendered as InputLabel and passed to InputLabel as children"